src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -1208,3 +1208,5 @@ private fun checkTypesAreSized(holder: RsAnnotationHolder, fn: RsFunction) {
         RsDiagnostic.SizedTraitIsNotImplemented(typeReference, ty).addToHolder(holder)
     }
 }
+
+private fun RsAttr.checkNameAndIsBuiltIn(target: String): Boolean = (metaItem.name == target && isBuiltInAttribute)
```

Comment:
Personaly, I'd merge it with `isBuiltInAttribute` and name it `isBuiltinWithName` (or something similar)

Reply:
Okay, I will merge it and move it to `RsAttr.kt`